### PR TITLE
Add kubeadm v1.8-branched jobs

### PR DESCRIPTION
**Pull request by luxas**
Fixes: https://github.com/kubernetes/kubeadm/issues/426

cc @kubernetes/sig-cluster-lifecycle-pr-reviews @krzyzacy @fejta @jdumars

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -8509,6 +8509,44 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-1-7-on-1-8": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-kubeadm-gce-1-7-on-1-8.env",
```
Comment (krzyzacy) [inline jobs/config.json]: the env files does not exist yet :-)

Comment (luxas) [reply jobs/config.json]: oh, yeah forgot to do that before pushing. No wonder it fails :)

Comment (krzyzacy) [reply jobs/config.json]: oh those are all empty files :-( let's just don't include those lines

Comment (luxas) [reply jobs/config.json]: May I clean these up in an upcoming PR?
I'd need to do the same for all other kubeadm jobs basically...
That way this PR is clean-ish :)

**Pull request merged**